main.tf

```diff
@@ -1269,6 +1275,315 @@ resource "juju_integration" "barbican-to-vault" {
   }
 }
 
+# Ironic feature.
+module "ironic" {
+  depends_on           = [module.single-mysql, module.many-mysql]
+  count                = var.enable-ironic ? 1 : 0
+  source               = "./modules/openstack-api"
+  charm                = "ironic-k8s"
+  name                 = "ironic"
+  model                = juju_model.sunbeam.name
+  channel              = var.ironic-channel == null ? var.openstack-channel : var.ironic-channel
+  revision             = var.ironic-revision
+  rabbitmq             = module.rabbitmq.name
+  mysql                = local.mysql["ironic"]
+  keystone             = module.keystone.name
+  keystone-cacerts     = module.keystone.name
+  ingress-internal     = juju_application.traefik.name
+  ingress-public       = juju_application.traefik-public.name
+  scale                = var.os-api-scale
```

Comment:
This also has been updated because of multi region support

Reply:
No longer relevant.